packages/manager/cypress/support/setup/delete-internal-header.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Intercept APIv4 responses to delete internal-only headers
+ * and ensure our tests replicate what users see.
+ */
+export const deleteInternalHeader = () => {
+  // Set up Linode APIv4 intercepts and set default alias value.
+  beforeEach(() => {
+    cy.wrap([]).as('linodeApiV4Request');
+    cy.intercept(
+      {
+        middleware: true,
+        url: /\/v4(?:beta)?\/.*/,
+      },
+      (req) => {
+        // Delete internal-only header
+        req.continue((res) => {
+          delete res.headers['akamai-internal-account'];
+        });
+      }
+    ).as('linodeApiV4Request');
```

Comment:
```suggestion
    cy.intercept(
      {
        middleware: true,
        url: /\/v4(?:beta)?\/.*/,
      },
      (req) => {
        // Delete internal-only header
        req.continue((res) => {
          delete res.headers['akamai-internal-account'];
        });
      }
    );
```
The aliases and that call to `cy.wrap` shouldn't be needed for this case (we used those in the other file just so we could get information about the requests after-the-fact, but no need for that here!)

Thanks Hussain!